vbfml/scripts/err_analysis

```diff
@@ -865,12 +864,12 @@ def plot_correl(ctx, input_dir: str) -> None:
 )
 @click.option(
     "-d",
-    "--dataset",
+    "--datasets",
     type=str,
-    default=("vbf_h", "ewk_v", "qcd_v"),
+    default=["vbf_h", "ewk_v", "qcd_v"],
```

Comment:
Are you sure that this work ?
because the multiple option return a tuple 
Just to make sure

Reply:
Thanks for checking, yeah I was trying it out to see if it gives an error somewhere if I do it, but it seems to work fine for me. Did you have an issue with it earlier?

Reply:
Just to make sure that this won't be an issue, I changed this into a `tuple` object in [cb294a9](https://github.com/alpakpinar/vbfml/pull/39/commits/cb294a9591fa94f7b4c388680a58d78c09215647). After all, we don't want to mutate this array anywhere right? So a `tuple` should be good for the job as well.

Reply:
All good then
I got a issue with lists, but it was earlier in the implementation and now I checked it is working with both tuples and lists so all good

Reply:
Great, thanks!